example/image-classification/symbols/mobilenetv2.py

```diff
@@ -17,60 +17,202 @@
+        if_act=False,
+        prefix='%s-linear'%prefix
+    )
+    if ifshortcut:
+        out = shortcut(
+            data_in=data,
+            data_residual=linear_out,
+            prefix=prefix,
+        ) 
+        return out
+    else:
+        return linear_out
 
-    Parameters
-    ----------
-    num_classes : int, default 1000
-        Number of classes for the output layer.
-    multiplier : float, default 1.0
-        The width multiplier for controling the model size. The actual number of channels
-        is equal to the original channel size multiplied by this multiplier.
-    ctx : Context, default CPU
-        The context in which to initialize the model weights.
-    """
-    net = MobileNetV2(multiplier=multiplier, classes=num_classes, **kwargs)
-    net.initialize(ctx=ctx, init=mx.init.Xavier())
-    net.hybridize()
+def inverted_residual_blocks(data, in_c, t, c, n, s, prefix):
+    first_block = inverted_residual_unit(
+        data=data,
+        num_in_filter=in_c,
+        num_filter=c,
+        ifshortcut=False,
+        stride=(s,s),
+        kernel=(3,3),
+        pad=(1,1),
+        expansion_factor=t,
+        prefix='%s-block0'%prefix
+    )
 
-    data = mx.sym.var('data')
-    out = net(data)
-    sym = mx.sym.SoftmaxOutput(out, name='softmax')
-    return sym
+    last_residual_block = first_block
+    last_c = c
 
+    for i in range(1,n):
+        last_residual_block = inverted_residual_unit(
+            data=last_residual_block,
+            num_in_filter=last_c,
+            num_filter=c,
+            ifshortcut=True,
+            stride=(1,1),
+            kernel=(3,3),
+            pad=(1,1),
+            expansion_factor=t,
+            prefix='%s-block%d'%(prefix, i)
+        )
+    return last_residual_block
 
-def plot_net():
-    """
-    Visualize the network.
-    """
-    sym = get_symbol(1000, prefix='mob_')
+MNETV2_CONFIGS_MAP = {
+    (224,224):{
+        'firstconv_filter_num': 32, # 3*224*224 -> 32*112*112
+        # t, c, n, s
+        'bottleneck_params_list':[
+            (1, 16, 1, 1), # 32x112x112 -> 16x112x112
+            (6, 24, 2, 2), # 16x112x112 -> 24x56x56
+            (6, 32, 3, 2), # 24x56x56 -> 32x28x28
+            (6, 64, 4, 1), # 32x28x28 -> 64x28x28
+            (6, 96, 3, 2), # 64x28x28 -> 96x14x14
+            (6, 160, 3, 2), # 96x14x14 -> 160x7x7
+            (6, 320, 1, 1), # 160x7x7 -> 320x7x7
+        ],
+        'filter_num_before_gp': 1280, # 320x7x7 -> 1280x7x7
+    } 
+}
 
-    # plot network graph
-    mx.viz.plot_network(sym, shape={'data': (8, 3, 224, 224)},
-                        node_attrs={'shape': 'oval', 'fixedsize': 'fasl==false'}).view()
+class MobileNetV2(object):
+    def __init__(self, data_wh, multiplier, **kargs):
+        super(MobileNetV2, self).__init__()
+        self.data_wh=data_wh
+        self.multiplier=multiplier
+        if self.data_wh in MNETV2_CONFIGS_MAP:
+            self.MNetConfigs=MNETV2_CONFIGS_MAP[self.data_wh]
```

Comment:
mixing Camel case with under score in names are not pretty, even though it is in example folder.